integration-tests/otlp/src/main/resources/otel-config.yaml

```
@@ -23,7 +23,7 @@ receivers:
           key_file: $MTLS_SERVER_KEY
 exporters:
   logging:
-    logLevel: $LOGGING_EXPORTER_LOG_lEVEL
+    loglevel: $LOGGING_EXPORTER_LOG_lEVEL
```

`logLevel` is actually [deprecated](https://github.com/open-telemetry/opentelemetry-collector/tree/main/exporter/loggingexporter#note). Should update this and [its usage](https://github.com/open-telemetry/opentelemetry-java/blob/main/integration-tests/otlp/src/main/java/io/opentelemetry/integrationtest/OtlpExporterIntegrationTest.java#L139) to `normal`. 
```suggestion
    verbosity: $LOGGING_EXPORTER_VERBOSITY_LEVEL
```

---

Done!